app/assets/javascripts/discourse/app/routes/discourse.js

```diff
@@ -5,12 +5,16 @@ import { service } from "@ember/service";
 import { seenUser } from "discourse/lib/user-presence";
 
 export default class DiscourseRoute extends Route {
-  @service router;
+  @service pmTopicTrackingState;
 
   willTransition() {
     seenUser();
   }
 
+  afterModel() {
+    this.pmTopicTrackingState.startTracking();
```

Comment:
I am not sure if this route is best for starting PM tracking, since admin routes extend from this one, too